index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GLOBAL SUMMIT 2022</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+        integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+</head>
+
+<body>
+    <div id='header' class="header">
+        <!-- Top nav -->
+        <div class="top-header">
+            <div class="container top-header-container">
+                <ul class='top-header-content'>
+                    <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#" class='login-btn'>Mypage</a></li>
+                    <li><a href="#" class='login-btn'>Logout</a></li>
+                </ul>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly hide this navbar in the mobile view like the previous reviewer asked and as provided in the guidelines.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190826980-a4b167ea-d5ad-4acc-8ce5-1314f87be5a2.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190826995-29dce62d-7369-4259-a969-8cec4f39ff6f.png)